frontend/src/components/Navbar/Navbar.jsx

```diff
@@ -1,51 +1,49 @@
-import Logo from "../Logo/Logo";
-import HomeButton from "../Buttons/HomeButton";
-import CreateNewLogButton from "../Buttons/CreateNewLogButton";
-import LogHistoryButton from "../Buttons/LogHistoryButton";
-import SignInButton from "../Buttons/SignInButton";
+import { NavLink } from 'react-router-dom';
 import { useAuth } from "../../contexts/AuthContext";
+import {
+  HomeIcon,
+  BookOpenIcon,
+  ClockIcon,
+  ArrowLeftStartOnRectangleIcon
+} from '@heroicons/react/24/outline';
 import "./Navbar.css";
-import SignOutButton from "../Buttons/SignOutButton";
 
-export default function Navbar({ variant }) {
-  return (
-    <div>
-      <NavbarComponent variant={variant} />
-    </div>
-  );
-}
-
-function NavbarComponent({ variant }) {
-  return (
-    <div className="navbar">
-      <Logo />
-      <NavButtons variant={variant} />
-    </div>
-  );
-}
-
-function NavButtons({ variant }) {
-  return (
-    <div className="nav-buttons-container">
-      {variant === "homepage" ? (
-        <Buttons />
-      ) : (
-        <>
-          <HomeButton />
-          <CreateNewLogButton variant="navbar" />
-          <LogHistoryButton variant="navbar" />
-        </>
-      )}
-    </div>
-  );
-}
+export const Navbar = () => {
+  const { logout } = useAuth();
 
-function Buttons() {
-  const { session } = useAuth();
+  const handleSignOut = async () => {
+    try {
+      await logout();
+    } catch {
+      console.log("Failed to logout");
+    }
+  };
 
   return (
-    <div className="nav-buttons-container">
-      {session ? <SignOutButton /> : <SignInButton />}
-    </div>
+    <nav className="nav-bar">
+      <div className="nav-button-container">
+        <NavLink className="nav-button" to="/homepage">
+          <div className="nav-button-icon-container">
+            <HomeIcon className="nav-icon"/>
+          </div>
+          <p className="nav-button-text">Home</p>
+        </NavLink>
+        <NavLink className="nav-button" to="/newLog">
+          <div className="nav-button-icon-container">
+            <BookOpenIcon className="nav-icon"/>
+          </div>
+          <p className="nav-button-text">Logs</p>
+        </NavLink>
+        <NavLink className="nav-button" to="/logHistory">
+          <div className="nav-button-icon-container">
+            <ClockIcon className="nav-icon"/>
+          </div>
+          <p className="nav-button-text">History</p>
+        </NavLink>
+      </div>
+      <button className="logout-button" onClick={handleSignOut}>
```

Comment:
I think this is more of a question for the designers. It might be best to wait for their input before making changes from the Figma design. What do you think?